src/plugins/validate-semantic/validators/2and3/schemas.js

```diff
@@ -23,41 +23,46 @@ export const validate2And3TypeArrayRequiresItems = () => (system) => {
     })
 }
 
+
+
 export const validate2And3TypesInDefaultValuesMatchesWithEnum = () => (system) => {
   return system.validateSelectors
     .allSchemas()
     .then(nodes => {
       return nodes.reduce((acc, node) => {
         const schemaObj = node.node
         const { type } = schemaObj || {}
+        const isNullable = schemaObj.nullable ? schemaObj.nullable : false
```

Comment:
```suggestion
        const isNullable = !!schemaObj.nullable
```

since `nullable` should always be a boolean, we can get away with coercing it directly!

Reply:
Done! Good advice!